cli/monitoring/advanced-model-monitoring.yaml

```diff
@@ -14,42 +14,43 @@ trigger:
     minutes: 15 # at 15 mins after 3am
 
 create_monitor:
+
   compute: 
     instance_type: standard_e4s_v3
-    runtime_version: 3.2
+    runtime_version: "3.3"
+
   monitoring_target:
     ml_task: classification
-    endpoint_deployment_id: azureml:fraud-detection-endpoint:fraud-detection-deployment
+    endpoint_deployment_id: azureml:credit-default:main
   
   monitoring_signals:
     advanced_data_drift: # monitoring signal name, any user defined name works
       type: data_drift
       # reference_dataset is optional. By default referece_dataset is the production inference data associated with Azure Machine Learning online endpoint
       reference_data:
         input_data:
-          path: azureml:my_model_training_data:1 # use training data as comparison reference dataset
+          path: azureml:credit-reference:1 # use training data as comparison reference dataset
           type: mltable
         data_context: training
-        target_column_name: fraud_detected
+        target_column_name: DEFAULT_NEXT_MONTH
```

Comment:
Should all the target_column_name fields be changed to data_column_names with target_column as a sub field? Asking to confirm whether is PR is meant for 1.14.0 February release